nemo/nemo/collections/nlp/data/language_modeling/megatron/gpt_dataset.py

```diff
@@ -321,21 +322,28 @@ def __init__(
             )
 
         super().__init__()
+        self.tokenizer = tokenizer
         self.name = name
         self.indexed_dataset = indexed_dataset
         self.drop_last = drop_last
         self.seq_length = seq_length
+        self.np_rng = np.random.RandomState(seed=seed) # rng state for FIM
```

Comment:
is seed same seed as specified in yaml?

Reply:
Hey @ptoulme-aws, indeed. Basically I'm extending MegatronGPTModel which passes `cfg.seed` during dataset creation.

https://github.com/aws-neuron/neuronx-nemo-megatron/blob/6038bb80b5ae59d3f4a99ed38a4f68fa0e22665a/nemo/nemo/collections/nlp/models/language_modeling/megatron_gpt_model.py#L876